editoast/editoast_authz/src/builtin_role.rs

```diff
@@ -12,6 +12,9 @@ pub enum BuiltinRole {
     InfraRead,
     #[strum(serialize = "infra:write")]
     InfraWrite,
+
+    #[strum(serialize = "speed_limit_tags:read")]
+    SpeedLimitTagsRead,
```

Comment:
Use RollingStockCollection* roles, it does't make much sense to work with speed limit tags while being denied access to rolling stocks.